src/pretix/plugins/sendmail/signals.py

```diff
@@ -19,5 +19,24 @@ def control_nav_import(sender, request=None, **kwargs):
             }),
             'active': (url.namespace == 'plugins:sendmail' and url.url_name == 'send'),
             'icon': 'envelope',
-        }
+            'children': True,
+            'second_level': [
```

Comment:
I like that we can do this now! Thanks! :)

Not sure it is necessary here, but let's keep it for now. One minor comment on this, though: There is no reason for two attributes, ``children`` and ``second_level``, that just creates room for errors. Just drop the boolean property and just go with ``'children': […]``. You can still to ``if nav.children`` then, it will still work.